CHANGELOG.md

```diff
@@ -132,7 +134,7 @@ be found.
 - Fix a race condition between device deferred removal and resume device, when using the devicemapper graphdriver [#23497](https://github.com/docker/docker/pull/23497)
 - Add `docker stats` support in Windows [#25737](https://github.com/docker/docker/pull/25737)
 - Allow using `--pid=host` and `--net=host` when `--userns=host` [#25771](https://github.com/docker/docker/pull/25771)
-+ (experimental) Add metrics output [#25820](https://github.com/docker/docker/pull/25820)
++ (experimental) Add metrics (Prometheus) output for basic `container`, `image`, and `daemon` operations [#25820](https://github.com/docker/docker/pull/25820)
```

Comment:
I wanted to add "Prometheus" in here, because I think people will look for that, but @crosbymichael if you could check if the description looks correct?